packages/router/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(warnings)]
```

Comment:
Since this package is very much WIP, I didn't see the point in allowing or fixing each instance individually. We'll need to remove this once the package is closer to ready.